ui/src/app/applications/components/application-details/application-details.scss

```diff
@@ -43,7 +43,7 @@
     }
 
     &__tab-content-full-height {
-        height: calc(100vh - 2 * 50px);
+        height: calc(100vh - 2 * 76px);
```

Comment:
Fixes scroll bug mentioned in https://github.com/argoproj/argo-cd/pull/5233#discussion_r562979174